ChangeHistory

```diff
@@ -45,6 +45,10 @@ ticket identifier, "G4CMP-nnn", or using the GitHub pull request, "PR-nn".
 2024-05-02  G4CMP-378 : Correct expression for phonon-QP scattering energy.
 2024-05-02  G4CMP-344 : Improve KaplanQP memory usage with mutable buffers.
 
+2024-05-07  G4CMP-404 : Improve charge carriers-phonons scattering.
+2024-08-16  G4CMP-120 : IV scattering process should include kinematic effects.
```

Comment:
Since you are requesting to merge onto the G4CMP-120 branch, you should update the date stamp to today.

Reply:
I have update the dated line of G4CMP-120 and G4CMP-427 to today

Reply:
Should I also update G4CMP-404 date?

Reply:
Nah, I think it's fine for now.  When your G4CMP-120 branch is ready to merge onto G4CMP-404, then you would set that datestamp.